ceph.spec.in

```diff
@@ -131,6 +131,7 @@ BuildRequires:	python-pecan
 BuildRequires:	python-requests
 BuildRequires:	python-virtualenv
 BuildRequires:	python-werkzeug
+BuildRequires:	socat
```

Comment:
Conditionalizing build dependencies is far beyond my poor-to-nonexistent Debian and FreeBSD packaging skills.